lib/fastlane/actions/mailgun.rb

```diff
@@ -26,13 +22,10 @@ def self.description
 
       def self.available_options
         [
-          FastlaneCore::ConfigItem.new(key: :mailgun_sandbox_domain,
```

Comment:
To add on to this... It might be a good idea to look for this old options in the action and warn the user they need to upgrade. The action suddenly failing might be a bit scary so a user may appreciate a nice littler warning/error/info/debug message :whale: